addons/common/config.cpp

```diff
@@ -5,7 +5,12 @@ class CfgPatches {
     units[] = {};
     weapons[] = {};
     requiredVersion = 0.1;
-    requiredAddons[] = {"fpa_main"};
+    requiredAddons[] = 
+	{
+		"fpa_main",
+		"A3_Data_F",
+		"A3_UI_f"
+	};

```

Comment:
These two above are not needed, fpa_main already requires full a3 + ace

Reply:
Done